spec/models/event_spec.rb

```diff
@@ -29,7 +29,7 @@
                             .with_options(limit: 75, null: false) }
 
     it { is_expected.to have_db_column(:image).of_type(:string)
-                            .with_options(limit: 225, null: false) }
+                            .with_options(limit: 225) }
```

Comment:
Expression at 32, 55 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 24) spaces for indenting an expression spanning multiple lines.